VirtoCommerce.Storefront.Model/Common/Money/Money.cs

```diff
@@ -369,25 +369,29 @@ public virtual string ToString(bool showDecimalDigits, bool showCurrencySymbol)
 
         /// <summary>
         /// Evenly distributes the amount over n parts, resolving remainders that occur due to rounding 
-        /// errors, thereby garuanteeing the postcondition: result->sum(r|r.amount) = this.amount and
+        /// errors, thereby garuanteeing the post condition: result->sum(r|r.amount) = this.amount and
         /// x elements in result are greater than at least one of the other elements, where x = amount mod n.
         /// </summary>
-        /// <param name="n">Number of parts over which the amount is to be distibuted.</param>
+        /// <param name="n">Number of parts over which the amount is to be distributed.</param>
```

Comment:
As proposed by @tatarincev, need to remove `Allocate` method at all as redudant

Reply:
done